dockerfiles/grist/Dockerfile

```diff
@@ -0,0 +1,31 @@
+ARG GRIST_VERSION=1.1.14
+
+FROM gristlabs/grist:$GRIST_VERSION
+
+WORKDIR /grist/static
+
+ARG LASUITE_VERSION=1.0.1
+ARG LASUITE_ARCHIVE=gouvfr-lasuite-integration-$LASUITE_VERSION.tgz 
+
+RUN apt-get update
+RUN apt-get install -y wget
+
+RUN wget https://github.com/numerique-gouv/lasuite-integration/releases/download/integration-v$LASUITE_VERSION/$LASUITE_ARCHIVE
```

Comment:
Remove purge wget after that

Reply:
Fixed in https://github.com/numerique-gouv/dockerfiles/pull/5/commits/dacc1f0675eef79c4728c6f262509478b9dd23f0